pkg/nonk8sapi/policies/list_status.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package policies
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stolostron/hub-of-hubs-manager/pkg/nonk8sapi/authentication"
+)
+
+// List Policies Status middleware.
+func ListStatus(authorizationURL string, authorizationCABundle []byte, dbConnectionPool *pgxpool.Pool) gin.HandlerFunc {
+	return func(ginCtx *gin.Context) {
+		user, isCorrectType := ginCtx.MustGet(authentication.UserKey).(string)
+		if !isCorrectType {
+			fmt.Fprintf(gin.DefaultWriter, "unable to get user from context")
+			user = "Unknown"
+		}
+
+		groups, isCorrectType := ginCtx.MustGet(authentication.GroupsKey).([]string)
+		if !isCorrectType {
+			fmt.Fprintf(gin.DefaultWriter, "unable to get groups from context")
+			groups = []string{}
+		}
+
+		fmt.Fprintf(gin.DefaultWriter, "got authenticated user: %v\n", user)
+		fmt.Fprintf(gin.DefaultWriter, "user groups: %v\n", groups)
+
+		query := sqlQuery()
+		fmt.Fprintf(gin.DefaultWriter, "query: %v\n", query)
+
+		handleRows(ginCtx, query, dbConnectionPool)
+	}
+}
+
+func sqlQuery() string {
```

Comment:
nit: can we change to a more clear name for what the function is doing.
Also I would suggest to make this a constant for better performance, since the function call need to build stack.